Initializer/Initialize.cs

```diff
@@ -14,23 +14,62 @@ public class Initialize
         private ManageAccessToEntity _manageAccessToEntity = new ManageAccessToEntity();
 
         private readonly string _pathRoot = Path.GetFullPath(Path.Combine(
-            Directory.GetCurrentDirectory(), @"..\..\..\..\LanguageSkills"));
+            Directory.GetCurrentDirectory(), @"..\..\..\..\LanguageSkills\wwwroot"));
         private bool _isData = true;
         private string _tempItem = "";
         private int _countStage = 0;
+        private List<Language> _allLanguages = new List<Language>();
 
-        private string _createPath(string directoryName, string fileName, string fileExtension)
+        private string _generatePath(string directoryName, string fileName, string fileExtension)
         {
-            return @"\wwwroot\Dictionary\" + directoryName + fileName + fileExtension;
+            //Generate a global path
+            string path = @"\Dictionary\" + directoryName + fileName + fileExtension;
+            return _isFileExist(path) ? path : 
+                _removeSpaces(@"\Dictionary\" + directoryName, fileName + fileExtension);
+        }
+
+        private bool _isFileExist(string path)
+        {
+            return File.Exists(_pathRoot + path);
+        }
+
+        private string _removeSpaces(string path, string fileName)
+        {
+            if (_isFileExist(path + fileName.Replace(" ", "")))
```

Comment:
Incorrect code - it may not work with file names that have spaces in the middle of file name.